from_cpython/Objects/stringobject.c

```diff
@@ -23,6 +23,22 @@ PyObject * _do_string_format(PyObject *self, PyObject *args, PyObject *kwargs) {
     return do_string_format(self, args, kwargs);
 }
 
+void _string_init() {
```

Comment:
Hi, @undingen our str implementation couldn't use `_PyUnicode_Init`(But I don't know the reason). @kmod suggest me add a initialization function and call `PyType_Ready` in it. FYI, the related discussion in Septemper 5th in Gitter. Maybe there has another way to do it?